doc/infrastructures.rst

```diff
@@ -50,17 +50,51 @@ It is strongly recommended to associate labels to your hosts. Labels allow to fi
+
+* ``label_identifier != "wanted_value"`` will match if the label with the given name has not ``wanted_value`` as a value. Example : ``somename != "somevalue"``
+
+
+Filter on numeric value
+^^^^^^^^^^^^^^^^^^^^^^^
+These filters are used to check how a label value compares to a numeric value. Numeric value is a number written without quotes and  an optional unit. Currently supported units are golang durations ("ns", "us" , "ms", "s", "m" or "h"), bytes units ("B", "KiB", "KB", "MiB",	"MB", "GiB", "GB", "TiB", "TB", "PiB", "PB", "EiB", "EB") and `International System of Units (SI) <https://en.wikipedia.org/wiki/Metric_prefix>`_. The case of the unit does not matter.  
+
+
+* ``label_identifier == wanted_value`` and ``label_identifier == wanted_value`` will match if the label with the given name has a value equal to ``wanted_value``. Example : ``somename = 100``
+* ``label_identifier != wanted_value`` will match if the label with the given name has a value different from ``wanted_value``. Example : ``somename != 100``
+* ``label_identifier > wanted_value`` will match if the label with the given name has a value strictly superior to ``wanted_value``. Example : ``somename > 100``
+* ``label_identifier < wanted_value`` will match if the label with the given name has a value strictly inferior to ``wanted_value``. Example : ``somename < 100``
+* ``label_identifier >= wanted_value`` will match if the label with the given name has a value superior or equal to ``wanted_value``. Example : ``somename >= 100``
+* ``label_identifier <= wanted_value`` will match if the label with the given name has a value  inferior or equal to ``wanted_value``. Example : ``somename <= 100``
+
+
+Filter on regex value
+^^^^^^^^^^^^^^^^^^^^^
+These filters are used to check if a label value contains or excludes a regex. Regex value has to be between simple or double quotes : ``""`` or ``''``. "Contains" means that the value (string) of the label contains at least one substring matching the regex. "Excludes" means that the value (string) of the label contains no substring matching the regex.
+
+* ``label_identifier ~= "wanted_value"``will match if the label with the given name has a value containing ``wanted_value``. Example : ``somename ~= "(a|bc)+"``
+
+* ``label_identifier !~ "wanted_value"`` will match if the label with the given name has a value excluding ``wanted_value`` . Example : ``somename !~ "(a|bc)+"``
+
+Filter on set appartenance
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+These filters are used to check is a label value is matching with one of the value of a set. 
 
-There are four kinds of filters supported by Yorc:
+* ``label_identifier in ("firstval", "secondval")`` will match if the label with the given name has for value ``firstval`` or  ``secondval``. Example : ``somename in ("gpu", "cpu", "none")`` 
```

Comment:
Are quotes mandatory here?
If yes examples of the next section should be updated.